spec/lib/traject/config/format_main_spec.rb

```diff
@@ -236,38 +236,11 @@
     end
   end
 
-  context 'online copy only of a Software/Multimedia and database' do
-    let(:record) do
-      MARC::Record.new.tap do |r|
-        r.leader = '02441cms a2200517 a 4500'
-        r.append(MARC::ControlField.new('008', '920901d19912002pauuu1n    m  0   a0eng  '))
-        r.append(MARC::DataField.new('999', ' ', ' ',
-                                     MARC::Subfield.new('a', 'INTERNET RESOURCE'),
-                                     MARC::Subfield.new('w', 'ASIS'),
-                                     MARC::Subfield.new('i', '2475606-5001'),
-                                     MARC::Subfield.new('l', 'INTERNET'),
-                                     MARC::Subfield.new('m', 'SUL'),
-                                     MARC::Subfield.new('t', 'DATABASE')))
-      end
-    end
-
-    xit 'is only a database' do
-      expect(result[field]).to eq ['Database']
-    end
-  end
-
   context 'both physical copy and online copy of a Software/Multimedia and database' do
     let(:record) do
       MARC::Record.new.tap do |r|
         r.leader = '02441cms a2200517 a 4500'
         r.append(MARC::ControlField.new('008', '920901d19912002pauuu1n    m  0   a0eng  '))
-        r.append(MARC::DataField.new('999', ' ', ' ',
-                                     MARC::Subfield.new('a', 'INTERNET RESOURCE'),
-                                     MARC::Subfield.new('w', 'ASIS'),
```

Comment:
Make this LC or Dewey instead, and keep the INTERNET RESOURCE part

Reply:
done